nginx/conf.d/dev.nginx.conf

```diff
@@ -1,10 +1,13 @@
+proxy_cache_path /etc/nginx/cache keys_zone=SC_CACHE:30m levels=1:2 inactive=30d max_size=1g;
```

Comment:
I think it would be better to create a named volume for the nginx cache rather than storing the data in the container and use an /opt/ path, compare how nginx-data-volume (which should be renamed to nginx-logs) is used in docker-compose.yml.

Reply:
Hey, Blake, I have added the nginx-caches volume in docker-compose, but I don't know how to apply it to the nginx configuration, can you help me see :)

https://github.com/suttacentral/suttacentral/pull/2247/commits/48ea320bb21863eec632ed79ba988ba32e97388b

Reply:
That should be fine.

Reply:
Thx Blake :)